package.json

```diff
@@ -28,6 +28,7 @@
     "eslint": "^4.0.0",
     "libxmljs": "^0.19.7",
     "mocha": "^5.0.0",
+    "rimraf": "^3.0.2",
```

Comment:
Popular node module used for deleting directories. I'm happy to use something else if you want though.